### DISCUSSION TITLE
Adding a page header

### DISCUSSION THREAD

**Discussion by PolarLava** [xenocrat/chyrp-lite#285] [General]
So I'm  migrating an old blog from Chyrp to  Chyrp Lite and looking to add a header to the page.  I had done this successfully in Chyrp, but doesn't seem to be working in Chyrp Lite.  Basically I'm including a file with the header content in it to the theme/<name>/layout/default.twig as such:
`\<!-- Header --\>
{% include "content/header.twig" %}

\<!-- Sidebar --\>
{% include "content/sidebar.twig" %}`

This seems to be where I would insert this, but when I do nothing  shows up in the page.  Anyone have any ideas?

**Comment by xenocrat** [comment]
Hello there,

You probably just need to delete the content of _includes/caches/twig_ to see the change. In productions mode, Chyrp Lite tells Twig to cache theme templates, and it does this very aggressively. If you want to make many changes to a theme and see the changes in real time, enable [debug mode](https://github.com/xenocrat/chyrp-lite/wiki/Debug-and-Tester-Modes).

The themes Blossom and Topaz `include` a sidebar, so you can look at them for a working example.

P.S. Thank you for making the original Categorize module. ^_^
